tools/runners/run-mongo.js

```diff
@@ -55,6 +55,11 @@ function spawnMongod(mongodPath, port, dbPath, replSetName) {
     // initializes faster. (Not recommended for production!)
     '--oplogSize', '8',
     '--replSet', replSetName,
+    // Avoid printing banner about optional free monitoring services.
+    // Previously this was accomplished by passing --quiet to the shell,
+    // because the --enableFreeMonitoring option was broken until version
+    // 4.0.3: https://jira.mongodb.org/browse/SERVER-36474
+    '--enableFreeMonitoring', 'off',
```

Comment:
Wow, on MongoDB 4.0.4 Linux this option is still not supported?
```
  1| Error parsing command line: unrecognised option '--enableFreeMonitoring'
  1| try '/home/ben/meteor/dev_bundle/mongodb/bin/mongod --help' for more information
  1| Unexpected mongo exit code 2. Restarting.
```
I don't want to blame malice, but these bugs do have the side-effect that more people will see their spam about the free monitoring service. 🤔

Reply:
Filed a bug: https://jira.mongodb.org/browse/SERVER-38862